x-pack/plugins/aiops/public/components/change_point_detection/use_change_point_agg_request.ts

```diff
@@ -187,11 +192,27 @@ export function useChangePointResults(
           runtimeMappings
         );
 
+        if (usageCollection?.reportUiCounter) {
+          usageCollection.reportUiCounter(
+            `run_via_${embeddingOrigin}`,
```

Comment:
I'm not sure the `run_via_` part is needed for the `appName` prop. Looking at other ui_counters, other examples are `dashboard`, `dataVisualizer`, `ml`. So we could just supply e.g. `dashboard` or `aiops` here.